### ISSUE TITLE
Accessibility requirements? 

### ISSUE THREAD

**Issue by alex-hall** [homedepot/infinite-wish-board#51]
What accessibility requirements does this application need? 

- ARIA compliant? 
- Is touch screen okay? Need physical keyboard?

**Comment by aclaire-thd** [comment]
Although this is UK Make-A-Wish, here's [some guidance](https://www.make-a-wish.org.uk/accessibility-policy/) on what might be relevant

**Comment by aclaire-thd** [comment]
Make-A-Wish does grant wishes related to physical disabilities (e.g. wheelchair ramp, accessible vehicle), so we'll want to consider that aspect of user interaction

**Comment by ForiDunk** [comment]
If you don't mind my input, here is a result of an Accessibility audit
[Make-A-Wish.pdf](https://github.com/homedepot/infinite-wish-board/files/3729598/Make-A-Wish.pdf)

**Comment by aclaire-thd** [comment]
Thanks for the info @ForiDunk ! I'm interested in the accessibility tooling you used for the audit & would love to have a PR for the fixes 👍

**Comment by ForiDunk** [comment]
@aclaire-thd if you open the project in Chrome Browser, you can open the Developer Tools and there you should have an 'Audit' tab. There you should see Google's lighthouse audit tool. It can check multiple stuff like SEO, Performance, Accessibility and so on...
More info you can find here: https://developers.google.com/web/tools/lighthouse/

**Comment by DemonFangs** [comment]
@aclaire-thd Let me know if this tickets is still open?

**Comment by blueocto** [comment]
> @aclaire-thd Let me know if this tickets is still open?

Considering how many PR's are still open from a year ago, I would assume it is! :-)
